apps/frontend-web/app/test/[id]/utils/index.ts

```diff
@@ -0,0 +1,43 @@
+import { TestInput, useApiClient } from "@test-assessment/cms-graphql-api";
+import { transformQuestionDuplicate } from "./helper";
+import { useRouter } from "next/navigation";
+
+type TestDetailUtils = {
+  onDuplicateTest: () => void;
+}
+
+export default function useTestDetail({ testId }): TestDetailUtils {
```

Comment:
update name with not args testId
should have loading to show loading for the Duplicate button